### DISCUSSION TITLE
Columns and Rows don't scroll when put inside other components like cards and containers

### DISCUSSION THREAD

**Discussion by mnobeidat13** [flet-dev/flet#726] [Q&A]
Why don't Columns and Rows scroll when put inside other components like cards and containers.
I set expand to True and scroll to 'hidden'

**Comment by ndonkoHenri** [comment]
Build up a sample code for testing please.

**Reply by gbrandt** [reply]
Columns don't scroll inside Rows as well: https://gist.github.com/gbrandt/c342ba045e551560a9236ed58974ae60

**Reply by ndonkoHenri** [reply]
@gbrandt you simply have to Add an expand to your container:
```python
Container(
           bgcolor=colors.AMBER,
           expand=True,
           content=Row(....
```
